axelor-supplychain/src/main/resources/views/Invoice.xml

```diff
@@ -15,6 +15,10 @@
     	<call class="com.axelor.apps.supplychain.web.InvoiceController" method="generateSubscriptionInvoices"/>
     </action-method>
     
+    <action-method name="action-supplychain-method-update-product-qty-with-pack-header-qty">
```

Comment:
Better to use name "action-invoice-method-update-product-qty-with-pack-header-qty"